lab/webapp/src/components/Dataset/components/DatasetMenu/components/Charts/BarChart/index.jsx

```diff
@@ -0,0 +1,203 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { Header, Loader } from 'semantic-ui-react';
+import Plot from 'react-plotly.js';
+
+class BarChart extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+    };
+
+    this.createPlotlyBarChart = this.createPlotlyBarChart.bind(this);
+    //this.createBarGraph = this.createBarGraph.bind(this);
+  }
+
+  componentDidMount() {
+    //this.createBarGraph();
+  }
+
+  // basic reference guide for plotly bar charts -
+  // https://plot.ly/javascript/bar-charts/
+  createPlotlyBarChart() {
+    const { valByRowObj, depCol } = this.props;
+    let testSet = [... new Set(valByRowObj[depCol])].sort();
+    let data_sorted = valByRowObj[depCol].sort();
+    let classCountObj = {};
+    let testData = {
+      x: [],
+      y: [],
+      type: 'bar'
+    };
+    let plotlyBarCharData = [];
+    data_sorted.forEach(val => {
+      classCountObj[val] = classCountObj[val] ? ++classCountObj[val] : 1;
+    })
+
+    // for every entry in depColSet, map keys to color
+    let colorObj = {};
+    let colorList = [];
+    let colorStringList = [
```

Comment:
Identical values for the color list in `colorStringList` and the color-to-name mapping variable are defined in both `BarChart/index.jsx` and `BarCharts/index.jsx`.  This color scheme should probably be defined once elseware and referenced from these components.

It's likely we will want to reference this color scheme for plots on other pages later.